src/OpenTelemetry.Exporter.Zipkin/README.md

```diff
@@ -27,6 +27,12 @@ You can configure the `ZipkinExporter` through
    associated with the telemetry has "service.name" defined, then it'll be
    preferred over this option.
 * `Endpoint`: URI address to receive telemetry (default `http://localhost:9411/api/v2/spans`).
+
+   The endpoint configuration can be provided either [via code](../../examples/Console/TestZipkinExporter.cs)
+    or
+   [via environment variable](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#zipkin-exporter),
```

Comment:
nit: my preference here is to explicitly list the env vars that are supported by the exporter: the spec can change before we update the env vars that are actually supported. See https://github.com/open-telemetry/opentelemetry-dotnet/pull/2123/files#diff-797b5292d89d2baa2c473633f23703a029fbeb2c4bfe86281512d6243d029a3eR55